work-anniversary/tests/test_integration_work_anniversary_employee.py

```diff
@@ -0,0 +1,21 @@
+"""
+Integration test between work anniversary lambda and employees service
+"""
+
+import os
+import sys
+import boto3
+from unittest import mock
+from unittest.mock import patch
+from work_anniversary import work_anniversary
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+@patch('work_anniversary.get_all_employees',return_value=[{'node':{'email':'test@qxf2.com','firstname':'Test','lastname':'Qxf2','dateJoined':'03-Feb-2013','isActive':'Y'}}])
+@patch('work_anniversary.credentials.PASSWORD', 'new_value') 
```

Comment:
we can use 'as' and we can shorten the lengthy usage of variables
Where are we using this 'new-value'